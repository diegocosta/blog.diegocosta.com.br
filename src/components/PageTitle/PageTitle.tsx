import styled from 'styled-components';

export default styled.h1`
  font-size: 22px;
  background: ${({ theme }) => theme.accentColor};
  display: inline;
  padding: 5px;
`;
