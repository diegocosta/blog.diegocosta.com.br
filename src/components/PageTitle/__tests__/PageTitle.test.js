import React from 'react';
import { render } from '@testing-library/react';

import PageTitle from '../PageTitle';

describe('<PageTitle />', () => {
  it('should render properly', () => {
    const { baseElement, queryByText } = render(<PageTitle>John Doe</PageTitle>);
    expect(queryByText('John Doe')).toBeTruthy();
    expect(baseElement).toMatchSnapshot();
  });
});
