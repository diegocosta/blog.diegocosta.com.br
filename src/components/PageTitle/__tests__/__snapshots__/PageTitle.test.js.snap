// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageTitle /> should render properly 1`] = `
.c0 {
  font-size: 22px;
  background: #d73738;
  display: inline;
  padding: 5px;
}

<body>
  <div>
    <h1
      class="c0"
    >
      John Doe
    </h1>
  </div>
</body>
`;
